### ISSUE TITLE
Application Button (Apply for Job) -> Website URL: Doesn't open in a new Window

### ISSUE THREAD

**Issue by knasmueller** [Automattic/WP-Job-Manager#1828]
This issue was already reported 2014 first time.

From my point of it´s common sense that outgoing links are opening in a new window. 
I please you to make this also happen for the Job Manager plugin. 
Thanks.
Robert

**Comment by Hastibe** [comment]
I think the only exception would be companies using WP Job Manager as an internal job board--they might not want outgoing links on the single job page to open in a new tab? For every other case, though, having outgoing links opening in new windows makes sense, in my opinion, as well. It would be very helpful to have this as an option!

@knasmueller, in case it is helpful, [here's a thread](https://wordpress.org/support/topic/way-to-have-company-website-link-open-in-new-tab/) I started on the WordPress support forum for the plugin, which provides instructions on how to modify a couple of the templates to have some of the links on the single job pages open in new windows.

**Comment by knasmueller** [comment]
Thank you for your comment and the link to the support forum. 
To change the core code directely is also a solution. But we want to avoid that because an update can overwrite that ...

**Comment by Hastibe** [comment]
You're welcome! The modifications that I linked to don't modify the core code, though, they change templates, which aren't overwritten when the plugin is updated ([see here](https://wpjobmanager.com/document/template-overrides/) for more details).

**Comment by knasmueller** [comment]
Thanks a lot!

**Comment by richardmtl** [comment]
You are correct that we previously had it added in 2014, and then it looks like we removed it. I don't see why, so I will leave this issue open for Developer input.

Template file: https://github.com/Automattic/WP-Job-Manager/blob/release/1.33.4/templates/job-application-url.php

**Comment by taliaShwartzBB** [comment]
am also waiting for this feature, please.

**Comment by mrsgigi71** [comment]
Hello, its 2025 - is there an ETA for this enhancement? Otherwise we are forced to move everything to another plugin...
